### PR TITLE
[DOCS-7100] Service Catalog Best Practices

### PR DESCRIPTION
**Pull request by brett0000FF**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by crunchypistachiobutter** [comment]
The content looks great! I don't think we need the /service_catalog/guides/best-practices section since the content is already included in 'using-the-application-field' and 'linking-infrastructure-telemetries' pages.

File: content/en/service_catalog/guides/best-practices.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+For microservices, a service typically coincides with a Kubernetes deployment because its a self-contained unit of functionality with well-defined ownership and other metadata. Therefore, other components in a microservice might be named automatically during the instrumentation process. Add an **application** field to all the automatically discovered components to group them with the core service.
+
+For monolithic services, defining multiple services might be helpful. At the minimum, you should choose a service to represent the monolith as a whole. Then, associate relevant metadata, like documentation or source code, and relevant telemetry, like pod metrics. It's sometimes useful to define additional services that represent other functional units within the monolith if they have separate ownership properties like operation runbooks and documentation. In cases where there's a clearly defined hierarchy between the monolith and other units within it, its recommended to use the **application** field in schema 2.1+. Set the **application** value as the service name for the monolith itself, and add this application field to all sub-services that belong to the monolith.
```
Comment (hestonhoffman) [inline content/en/service_catalog/guides/best-practices.md:22]:
```suggestion
For monolithic services, defining multiple services might be helpful. At the minimum, you should choose a service to represent the monolith as a whole. Then, associate relevant metadata, like documentation or source code, and relevant telemetry, like pod metrics. It's sometimes useful to define additional services that represent other functional units within the monolith if they have separate ownership properties like operation runbooks and documentation. In cases where there's a clearly defined hierarchy between the monolith and other units within it, it's recommended to use the **application** field in schema 2.1+. Set the **application** value as the service name for the monolith itself, and add this application field to all sub-services that belong to the monolith. 
```

**Review by hestonhoffman — APPROVED** [review]
Tiny edit. LGTM

File: content/en/service_catalog/guides/best-practices.md (hunk 1 above)
Comment (brett0000FF) [reply content/en/service_catalog/guides/best-practices.md]: Oof, nice catch! 👀  Thanks!

**Comment by crunchypistachiobutter** [comment]
LGTM!

**Pull request merged**